code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -612,6 +611,12 @@
 			B.disguise (A)
 		else
 			to_chat(src, "<span class='danger'><B>Your powers are on cooldown! You must wait 20 seconds between bombs.</B></span>")
+			
+/mob/living/simple_animal/hostile/guardian/bomb/verb/Detonate
```

Comment:
Believe me, compiling and testing your code is important af. I, once,
managed to kill BestRP for a day with some untested code. Never again.